pkg/services/sqlstore/migrations/ualert/tables.go

```diff
@@ -492,9 +492,6 @@ func addAlertImageMigrations(mg *migrator.Migrator) {
 }
 
 func extractAlertmanagerConfigurationHistoryMigration(mg *migrator.Migrator) {
-	if !mg.Cfg.UnifiedAlerting.IsEnabled() {
```

Comment:
This always ran before the migration, so it never did anything to new
legacy migrations anyways.